packages/modules/src/interfaces/IValidationModule.ts

```diff
@@ -0,0 +1,12 @@
+import { Signer } from 'ethers'
+import { UserOperation } from '@biconomy/core-types'
+import { Bytes } from 'ethers/lib/utils'
+
+export interface IValidationModule {
+  getAddress(): Promise<string>
+  getInitData(): Promise<string>
```

Comment:
What is the use of getInitData() in IValidationModule? 
It should be in SA which knows about factory address and the validation module address which needs to be used as default validation module.